src/open_inwoner/accounts/backends.py

```diff
@@ -314,3 +314,183 @@ def update_user(self, user: AbstractUser, claims: JSONObject):
+      facility for EIDAS users to connect their EIDAS identity to a BSN, which we then
+      may or may not have.
+    """
+
+    OIP_UNIQUE_ID_USER_FIELDNAME = dynamic_setting[Literal["eidas_pseudo_id"]]()
+    OIP_LOGIN_TYPE = dynamic_setting[LoginTypeChoices]()
+    config_class = OpenIDEIDASConfig
+
+    # TODO: Can probably remove
+    def _check_candidate_backend(self) -> bool:
+        parent = super()._check_candidate_backend()
+        return parent and self.config_class is OpenIDEIDASConfig
+
+    def _extract_eidas_claim_values(self, claims: JSONObject) -> EIDASClaimValues:
+        """
+        Extract all eIDAS claim values from the claims dict.
+
+        Returns a typed dictionary with all possible claim values (or None if not present).
+        """
+        eidas_config = cast(OpenIDEIDASConfig, self.config_class.get_solo())
+
+        return EIDASClaimValues(
+            pseudo_id=glom(
+                claims,
+                Path(*eidas_config.pseudo_identifier_claim),
+                default=None,
+            ),
+            bsn=glom(
+                claims,
+                Path(*eidas_config.natural_person_bsn_identifier_claim),
+                default=None,
+            ),
+            company_name=glom(
+                claims,
+                Path(*eidas_config.company_name_claim),
+                default=None,
+            ),
+            legal_entity_id=glom(
+                claims,
+                Path(*eidas_config.legal_entity_identifier_claim),
+                default=None,
+            ),
+            first_name=glom(
+                claims,
+                Path(*eidas_config.natural_person_first_name_claim),
+                default=None,
+            ),
+            family_name=glom(
+                claims,
+                Path(*eidas_config.natural_person_family_name_claim),
+                default=None,
+            ),
+        )
+
+    def _construct_user_creation_kwargs_from_claims(
+        self, claims: JSONObject
+    ) -> dict | None:
+        """
+        Extract user filter and create kwargs based on eIDAS claim patterns.
+        """
+        # TODO: Remove before merging
+        logger.info("extract EIDAS OIDC claims", claims=claims)
+
+        # Extract all claim values
+        claim_values = self._extract_eidas_claim_values(claims)
+        logger.info("eidas_oidc_claims_received", claim_keys=list(claims.keys()))
+
+        # Validate presence of requried pseudo_id
+        if not claim_values["pseudo_id"]:
+            msg = (
+                "Claims did not include a value for the configured pseudo identifier claim. "
+                "Pseudo identifier is required for all eIDAS authentications."
+            )
+            self.log_system_error(msg)
+            raise SuspiciousOperation(msg)
+
+        # Validate that at least one identifying claim is present (excluding pseudo_id)
+        identifying_claims = {k: v for k, v in claim_values.items() if k != "pseudo_id"}
```

Comment:
On the one hand, the docstring says "WE rely on the Pseudo ID, which is guaranteed to be unique per service provider/user". But here you raise an error if the identifying claims consist only of `pseudo_id`'s. Is this a contradiction? I guess I don't fully grok the purpose and concept of `pseudo_id`.